### ISSUE TITLE
xPrivacy on LP not working with boot.img changes applied

### ISSUE THREAD

**Issue by wsoyka-tgm** [M66B/XPrivacy#2154]
Device: OnePlus One (Bacon)
Rom: Sabermod CM12

After reflashing the boot.img with changes applied as advised in the readme, reflashing Xposed, reinstalling Xposed installer, and then reinstalling the provided xPrivacy APK for Lollipop, aaand then activating the module and rebooting - xPrivacy still shoots the "xPrivacy is not enabled in the Xposed installer" notification and is not working. Other Xposed modules work.

tl;dr: xPrivacy doesn't work on Bacon, Lollipop with "xprivacy453 u:object_r:system_server_service:s0" apended to service_contexts

**Comment by M66B** [comment]
Please provide a logcat.
This may be caused by a known Xposed problem.
Possibly duplicate of #2148

**Comment by wsoyka-tgm** [comment]
Logcat: http://pastebin.com/xBaCvM88
(I did crop the Logcat, as it was both redundant and also too large for pastebin, let me know if i cropped something important/should upload the rest aswell)

**Comment by wsoyka-tgm** [comment]
Also, can you tell/link me what problem of Xposed you mean, so I and possibly others with the same issue can keep an eye on it?

**Comment by M66B** [comment]
Small methods are sometimes optimized and cannot be hooked by Xposed. Typical methods which are hooked to return true to let the UI part of a module query if the module part is active.
